File: Easy/onboarding.php
<?php
/**
 * CodinGame planet is being attacked by slimy insectoid aliens.
 * <---
 * Hint:To protect the planet, you can implement the pseudo-code provided in the statement, below the player.
 **/


// game loop
while (TRUE)
{
    // $enemy1: name of enemy 1
    fscanf(STDIN, "%s", $enemy1);
    // $dist1: distance to enemy 1
    fscanf(STDIN, "%d", $dist1);
    // $enemy2: name of enemy 2
    fscanf(STDIN, "%s", $enemy2);
    // $dist2: distance to enemy 2
    fscanf(STDIN, "%d", $dist2);

    if($dist1 < $dist2){
        echo("$enemy1\n");
    }
    else{
        echo("$enemy2\n");
    }

    // You have to output a correct ship name to shoot ("Buzz", enemy1, enemy2, ...)
}
?>

File: Easy/onboarding.vb
Module Player
' CodinGame planet is being attacked by slimy insectoid aliens.
' <---
' Hint:To protect the planet, you can implement the pseudo-code provided in the statement, below the player.

    Sub Main ()
        
        ' game loop
        While True
            Dim enemy1 as String
            enemy1 = Console.ReadLine() ' name of enemy 1

            Dim dist1 as Integer
            dist1 = Console.ReadLine() ' distance to enemy 1

            Dim enemy2 as String
            enemy2 = Console.ReadLine() ' name of enemy 2

            Dim dist2 as Integer
            dist2 = Console.ReadLine() ' distance to enemy 2

            if(dist1 < dist2)
                Console.WriteLine(enemy1)
            else
                Console.WriteLine(enemy2)
            end if
        End While
    End Sub
End Module
